### PR TITLE
Fix some typo and grammar

### PR DESCRIPTION
**Pull request by 928234269**
Signed-off-by: Sakura <longfei.shang@daocloud.io>

**What this PR does / why we need it**:
This PR corrected grammar in docs/charts.md and fixes a simple typo in pkg/storage/storage.go, pkg/kube/client.go
**Special notes for your reviewer**:

**If applicable**:
- [x] this PR contains documentation

### CODE REVIEW

File: pkg/proto/hapi/release/status.pb.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ const (
 	Status_PENDING_INSTALL Status_Code = 6
 	// Status_PENDING_UPGRADE indicates that an upgrade operation is underway.
 	Status_PENDING_UPGRADE Status_Code = 7
-	// Status_PENDING_ROLLBACK indicates that an rollback operation is underway.
+	// Status_PENDING_ROLLBACK indicates that a rollback operation is underway.
```
Comment (thomastaylor312) [inline pkg/proto/hapi/release/status.pb.go]: Was this comment in the actual protobuf file as well? This file is autogenerated from that one

Comment (928234269) [reply pkg/proto/hapi/release/status.pb.go]: yes, protobuf file has been updated.

**Comment by thomastaylor312** [comment]
@928234269 Should be good to go once you sign your latest commit for the DCO

**Comment by 928234269** [comment]
> @928234269 Should be good to go once you sign your latest commit for the DCO

This pr has been closed and I recreated a pr because of a signature issue. #6259

**Pull request closed without merging**